packages/adders/vitest.config.ts

```diff
@@ -7,10 +7,10 @@ export default defineConfig({
 	test: {
 		name: 'adders',
 		include: ['_tests/**/test.{js,ts}'],
+		globalSetup: ['_tests/_setup/global.ts'],
 		testTimeout: ONE_MINUTE * 3,
 		hookTimeout: ONE_MINUTE * 3,
-		maxConcurrency: 10,
-		globalSetup: ['_tests/_setup/global.ts'],
-		retry: env.CI ? 3 : 0
+		retry: env.CI ? 3 : 0,
+		pool: 'threads'
```

Comment:
I know, you are going to love this. This causes problems on windows :D.

Running the tests this line produces the following error for basically all the tests:
```
TypeError: process.chdir() is not supported in workers
 - /node_modules/.pnpm/playwright-core@1.48.2/node_modules/playwright-core/lib/utilsBundleImpl/index.js:43:461
 - /node_modules/.pnpm/playwright@1.48.2/node_modules/playwright/lib/common/expectBundleImpl.js:45:6553
 - /node_modules/.pnpm/degit@2.8.4/node_modules/degit/dist/index-688c5d50.js:7940:9
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:207:20
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:212:12
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:270:14
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:277:13
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:298:26
 - /node_modules/.pnpm/tinyexec@0.3.1/node_modules/tinyexec/dist/main.js:544:83
```

So we either have to remove all those tools from the testing happy path, or just remove that line

Reply:
how lovely. will fix momentarily